SUMMARY.md

```diff
@@ -1,24 +1,28 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/INSTALLATION.md)
+  * [Docker Installation](installation/INSTALLATION-DOCKER.md)
+  * [Local Installation](installation/INSTALLATION-LOCAL.md)
 * [Backend](backend/README.md)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-24T13:22:51Z" title="Wednesday, April 24th 2019, 3:22:51 pm +02:00">Apr 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,28 +1,34 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/README.md)
+  * [Docker Installation](installation/installation-docker.md)
+  * [Local Installation](installation/installation-local.md)
 * [Backend](backend/README.md)
```


When I visit https://docs.human-connection.org/human-connection/~/revisions/-LdEQ_-L4kwVw-mn1ibR/v/docs-improve_installation_instructions/backend which is the backend overview page, I see a blank page with links to the subpages.

![Screenshot (86)](https://user-images.githubusercontent.com/2110676/56662659-ce1a2380-66a4-11e9-96dc-6134b7197388.png)